packages/cli/lib/validators/marketplaceValidators/AbsoluteValidator.js

```diff
@@ -2,23 +2,23 @@ const path = require('path');
 
 const { VALIDATION_RESULT } = require('../constants');
 
-class BaseValidator {
+class AbsoluteValidator {
```

Comment:
This works for me 👍 . One other potential solution is to pass something like "useRelativePath" as a config option to the constructor for the `BaseValidator`. Definitely not a blocker though